components/lib/crash/docs/metrics.md

```diff
@@ -17,7 +17,7 @@ The following metrics are added to the ping:
 
 | Name | Type | Description | Data reviews | Extras | Expiration |
 | --- | --- | --- | --- | --- | --- |
-| crash_metrics.crash_count |[labeled_counter](https://mozilla.github.io/glean/book/user/metrics/labeled_counters.html) |Counts the number of crashes that occur in the application.  This measures only the counts of each crash in association with the labeled type of the crash. The labels correspond to the types of crashes handled by lib-crash.  |[1](https://bugzilla.mozilla.org/show_bug.cgi?id=1553935#c3)|<ul><li>uncaught_exception</li><li>caught_exception</li><li>native_code_crash</li></ul>|never |
+| crash_metrics.crash_count |[labeled_counter](https://mozilla.github.io/glean/book/user/metrics/labeled_counters.html) |Counts the number of crashes that occur in the application.  This measures only the counts of each crash in association with the labeled type of the crash. The labels correspond to the types of crashes handled by lib-crash.  |[1](https://bugzilla.mozilla.org/show_bug.cgi?id=1553935#c3)|<ul><li>uncaught_exception</li><li>caught_exception</li><li>fatal_native_code_crash</li><li>nonfatal_native_code_crash</li></ul>|never |
```

Comment:
Looks like the docs where not updated: any chance you could build so that this file gets updated?

Reply:
Sure, thanks for catching this.